src/libraries/Microsoft.PowerFx.Core/Binding/BinderUtils.cs

```diff
@@ -254,11 +255,22 @@ internal static TexlFunction FindBestErrorOverload(TexlFunction[] overloads, DTy
 
             // Consider overloads that have DType.Error parameter the last
             candidates = candidates.OrderBy(candidate => candidate.ParamTypes.Length > 0 && candidate.ParamTypes[0] == DType.Error).ToArray();
+
             foreach (var candidate in candidates)
             {
-                if (candidate.ParamTypes.Length > 0 && candidate.ParamTypes[0].Accepts(argTypes[0], exact: true, useLegacyDateTimeAccepts: false, usePowerFxV1CompatibilityRules: usePowerFxV1CompatibilityRules))
+                if (candidate.ParamTypes.Length > 0 && candidate.ParamTypes.Length >= cArg)
                 {
-                    return candidate;
+                    var bestCandidate = true;
+
+                    for (int i = 0; i < cArg; i++)
+                    {
+                        bestCandidate = bestCandidate && candidate.ParamTypes[i].Accepts(argTypes[i], exact: true, useLegacyDateTimeAccepts: false, usePowerFxV1CompatibilityRules: usePowerFxV1CompatibilityRules);
+                    }
+
+                    if (bestCandidate)
+                    {
+                        return candidate;
+                    }
                 }
```

Comment:
Patch has 5 overloads and binder was suggesting a wrong one when asking for intellisense suggestions.